src/costab.c

```diff
@@ -372,15 +372,15 @@ static Obj FuncMakeConsequences(Obj self, Obj list)
 {
     Obj                 hdSubs;         /*                                 */
     Obj                 objRels;        /*                                 */
-    Obj *               ptRel;          /* pointer to the relator bag      */
-    Obj *               ptNums;         /* pointer to this list            */
-    Int                 lp;             /* left pointer into relator       */
-    Int                 lc;             /* left coset to apply to          */
-    Int                 rp;             /* right pointer into relator      */
-    Int                 rc;             /* right coset to apply to         */
-    Int                 tc;             /* temporary coset                 */
-    Int                 i;              /* loop variable                   */
-    Obj                 hdTmp;          /* temporary variable              */
+    Obj *               ptRel;          // pointer to the relator bag
+    Obj *               ptNums;         // pointer to this list
+    Int                 lp;             // left pointer into relator
+    Int                 lc;             // left coset to apply to
+    Int                 rp;             // right pointer into relator
+    Int                 rc;             // right coset to apply to
+    Int                 tc;             // temporary coset
+    Int                 i;              // loop variable
+    Obj                 hdTmp;          // temporary variable
 
     /*T 1996/12/03 fceller this should be replaced by 'PlistConv'          */
```

Comment:
Aha, indeed. This is because of the `T`. We have to decide how to treat such comments. Presumably could be converted to `// TODO: `. Does that seems sensible?

Just to explain, the main part of this (in the first commit) was created using a very simple perl invocation. The idea is to take care of the 95% of "easy" cases in a fully automated way, to make it easy to rebase the PR if needed. Then the remaining cases may need some more manual work. Of course I can add more "automated" changes, but I'd like to keep each separate transformation as simple as possible.